src/main/java/springboot/Model/Instructor.java

```diff
@@ -1,13 +1,8 @@
-package Model;
+package springboot.Model;
 
-import javax.persistence.CascadeType;
-import javax.persistence.Entity;
-import javax.persistence.OneToMany;
-import javax.persistence.OneToOne;
-import java.util.ArrayList;
-import java.util.List;
+import javax.persistence.*;
 
-@Entity
+//@Entity
```

Comment:
```suggestion

```

Reply:
Are we adding this or just leaving it as a comment?